receiver/k8seventsreceiver/factory.go

```diff
@@ -44,6 +45,7 @@ func createDefaultConfig() config.Receiver {
 		APIConfig: k8sconfig.APIConfig{
 			AuthType: k8sconfig.AuthTypeServiceAccount,
 		},
+		InitialLookback: 60 * time.Second,
```

Comment:
Just curious, but why not just `time.Minute`?

Reply:
Same as above.